pedialabsnew/settings_shared.py

```diff
@@ -31,15 +31,12 @@
     'pedialabsnew.main',
     'pedialabsnew.exercises',
     'pedialabsnew.rstplot',
-    'django_markwhat',
 ]
 
 PAGEBLOCKS = [
     'pageblocks.TextBlock',
     'pageblocks.HTMLBlock',
     'pageblocks.PullQuoteBlock',
-    'pageblocks.ImageBlock',
```

Comment:
These pageblocks don't need to be removed once we update sorl to version 3.4.

Reply:
Even with the new sorl package ImageBlock and ImagePullQuoteBlock are still throwing errors: `LookupError: App 'pageblocks' doesn't have a 'Image(PullQuote)Block' model.`

Reply:
Interesting - Natalia or I will take a look here.

Reply:
Okay - this is something I messed up when making a release of django-pageblocks years ago. I removed these pageblocks because of these sorl issues: https://github.com/ccnmtl/django-pageblocks/releases/tag/2.0.0

But, it turns out that a better way to modernize this was to just address the issues directly in sorl.


Long story short, version 2.0.0 of django-pageblocks is just.. a branch that should not be used unless it's for a new application. We haven't used django-pageblocks in any new applications for a while.

Reply:
So the steps forward here are to downgrade to django-pageblocks 1.2.0. There likely are changes that need to be made to this branch of django-pageblocks, and we can release a new version 1.2.1 containing these fixes.

Reply:
At the very least, let's make the downgrade to django-pageblocks 1.2.0 here. I think we don't want to use version 2.0.0.

Then, if there are failing tests with 1.2.0, we will address those issues in this library as needed, with a 1.2.1 release.

Reply:
Done